src/drivers/cloudflare-kv-binding.ts

```diff
@@ -18,8 +18,18 @@ export default defineDriver((opts: KVOptions) => {
   async function getKeys(base: string = "") {
     base = r(base);
     const binding = getKVBinding(opts.binding);
-    const kvList = await binding.list(base ? { prefix: base } : undefined);
-    return kvList.keys.map((key) => key.name);
+    const keys = [];
+    let cursor: string | undefined = undefined;
+    do {
+      const kvList = await binding.list({ prefix: base || undefined, cursor });
+
+      keys.push(...kvList.keys.map((key) => key.name));
```

Comment:
perf: i think we can do the map in last `return` line

Reply:
Updated